lib/hiera/interpolate.rb

```diff
@@ -59,9 +64,19 @@ def do_interpolation(data, scope, extra_data, context)
     end
     private :do_interpolation
 
-    def get_interpolation_method_and_key(data)
+    def get_interpolation_method_and_key(data, is_config)
```

Comment:
Please note that both `do_interpolation` and `get_interpolation_method_and_key` are private methods.